src/rgw/setup/rgw.py

```diff
@@ -187,36 +187,78 @@ def cleanup(conf: MappedConf, pre_factory: bool = False):
         return 0
 
     @staticmethod
-    def upgrade(conf: MappedConf):
+    def _update_rgw_property(conf: MappedConf, config_index:str, config_key: str, config_val: str):
```

Comment:
why its called _update_rgw\_**property**() and not _update_rgw\_**config**().

same for remove

Reply:
done